docs/source/bouncing.rst

```diff
@@ -0,0 +1,136 @@
+How PaaSTA Bounces New Code and SOA-config changes
+==================================================
+
+In the context of this document, "Bouncing" refers to the procedure of
+replacing old tasks with new ones. With long-running services bouncing
+is implemented by controlling Marathon Apps. For periodic tasks, see the
+section on "Chronos Bouncing."
+
+A "Bounce" can happen for one of these reasons:
+
+* A new version of the code is deployed (a git sha change)
+* A change of `soa-configs <yelpsoa_configs.html>`_ for a service. (Change in ram, cpu, environment variables, etc)
```

Comment:
Do we want to duplicate the auto-generated note about which fields will not trigger a bounce?